lib/ApplicationFeatures/LanguageFeature.cpp

```diff
@@ -176,13 +177,46 @@ void* LanguageFeature::prepareIcu(std::string const& binaryPath,
   return icuDataPtr;
 }
 
+void LanguageFeature::prepareTimeZoneData(std::string const& binaryPath,
```

Comment:
I rather meant copying the LanguageFeature => TimezoneFeature than adding that functionality here.

Reply:
done